app/controllers/application_controller.rb

```diff
@@ -31,11 +32,19 @@ def set_locale
 
 
   def render_not_found
-    raise ActionController::RoutingError.new('Not Found')
+    raise ActionController::RoutingError.new('Not found')
```

Comment:
This method is not useful anymore. See my next comments. 

It also can be just replaced by `not_found`